autometa/common/coverage.py

```diff
@@ -171,15 +224,22 @@ def main(args):
         format='%(asctime)s : %(name)s : %(levelname)s : %(message)s',
         datefmt='%m/%d/%Y %I:%M:%S %p',
         level=logger.DEBUG)
-    parser = argparse.ArgumentParser('Autometa Coverage')
+    parser = argparse.ArgumentParser(description='Constuct contig coverage table given an input assembly and reads.')
```

Comment:
I interpreted issue-#21 item 1 as the main script description here.

Reply:
Yes, the same description is also at the top of the script in a comment.